CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+## v1.7.0 (06/15/2020)
+
+**Common**
+
+* Fixed various style issues.
+* Fixed a case when TaskContext could trigger cancellation after a successful operation.
+* `default_cache_expiration` added to `olp::client::OlpClientSettings`. Now users can control the time how long the downloaded data is considered valid.
+* Added `OLP_SDK_DISABLE_DEBUG_LOGGING` CMake option. When enabled, the SDK does not print TRACE/DEBUG messages.
+* Added `compression` setting to `olp::cache::CacheSettings`. The flag is passed to the leveldb and used to enable or disable the snappy compression.
+* The `olp::cache::DefaultCache` read operation changed. As a result, the number of memory allocation decreased, performance increased.
+* A new request type `OPTIONS` is added.
+* The LRU eviction now evicts expired elements first. The eviction time decreased.
+* Removed the retry code from `olp::http::NetworkCurl`, the retry is performed by `olp::client::OlpClient`.
+* Added a prewarm helper method to `olp::client::OlpClientSettingsFactory`, used to perform dns-prefetch.
```

Comment:
```suggestion
* Added a prewarm helper method to `olp::client::OlpClientSettingsFactory`. This can be used to perform DNS prefetch and TLS preconnect for certain URLs that are widely used by the clients, e.g. account and lookup.
```